media-sound/frescobaldi/frescobaldi-3.1.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
+inherit distutils-r1 gnome2-utils xdg-utils
```

Comment:
please drop gnome2-utils and switch to the equivalent xdg-utils functions in pkg* phases.

Reply:
Done.